docs/data/date-pickers/adapters-locale/AmPMCustomization.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import dayjs from 'dayjs';
+import {} from 'dayjs/locale/de';
+import {} from 'dayjs/locale/en-gb';
+import Stack from '@mui/material/Stack';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { Unstable_NextTimePicker as NextTimePicker } from '@mui/x-date-pickers/NextTimePicker';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+
+const locales = ['en', 'en-gb', 'de'];
+
+type LocaleKey = typeof locales[number];
+
+export default function AmPMCustomization() {
+  const [locale, setLocale] = React.useState<LocaleKey>('en');
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs} adapterLocale={locale}>
+      <Stack spacing={3}>
```

Comment:
🙈 
![Screenshot 2022-11-23 at 09 36 39](https://user-images.githubusercontent.com/4941090/203493286-f3ef3359-5127-4716-b32c-6fe409596d14.png)

```suggestion
      <Stack spacing={3} width={200}>
```